### ISSUE TITLE
Unable to execute bitbake

### ISSUE THREAD

**Issue by jwyune** [STMicroelectronics/meta-azureiotedge#1]
Dear ST,

I followed the instruction described [here](https://github.com/STMicroelectronics/meta-azureiotedge#installation). However, I cannot bitbake an image. Note that I am using **Thud**. How can I reproduce the same result that you have?

```
jiwon@sdt_dev:~/workspace/STM32MPU/Distribution-Package/azure/openstlinux-4.19-thud-mp1-19-10-09$ DISTRO=openstlinux-weston MACHINE=stm32mp1 source layers/meta-st/scripts/envsetup.sh
[HOST DISTRIB check]
Linux Distrib: Ubuntu
Linux Release: 18.04

[source layers/openembedded-core/oe-init-build-env][with previous config]

===========================================================================
Configuration files have been created for the following configuration:

    DISTRO            :  openstlinux-weston
    DISTRO_CODENAME   :  thud
    MACHINE           :  stm32mp1
    BB_NUMBER_THREADS :  <no-custom-config-set>
    PARALLEL_MAKE     :  <no-custom-config-set>

    BUILDDIR          :  build-openstlinuxweston-stm32mp1
    DOWNLOAD_DIR      :  <no-custom-config-set>
    SSTATE_DIR        :  <no-custom-config-set>

    SOURCE_MIRROR_URL :  <no-custom-config-set>
    SSTATE_MIRRORS    :  <no-custom-config-set>

    WITH_EULA_ACCEPTED:  YES

===========================================================================

Available images for OpenSTLinux layers are:

  - Official OpenSTLinux images:
      st-image-weston       -   OpenSTLinux weston image with basic Wayland support (if enable in distro)

  - Other OpenSTLinux images:
      - Supported images:
          st-image-core         -   OpenSTLinux core image
      - Proposed images as example only:
          st-example-image-qt   -   ST example of image based on QT framework   (require 'openstlinux-eglfs' distro)
          st-example-image-x11  -   ST example of image based on X11            (require 'openstlinux-x11' distro)
          st-example-image-xfce -   ST example of image based on XFCE framework (require 'openstlinux-x11' distro)
          and more images are available on meta-st-openstlinux/recipes-samples/images.

You can now run 'bitbake <image>'

jiwon@sdt_dev:~/workspace/STM32MPU/Distribution-Package/azure/openstlinux-4.19-thud-mp1-19-10-09/build-openstlinuxweston-stm32mp1$ bitbake st-image-weston
NOTE: Started PRServer with DBfile: /home/jiwon/workspace/STM32MPU/Distribution-Package/azure/openstlinux-4.19-thud-mp1-19-10-09/build-openstlinuxweston-stm32mp1/cache/prserv.sqlite3, IP: 127.0.0.1, PORT: 42031, PID: 24217
Loading cache: 100% |#################################################################################| Time: 0:00:00
Loaded 3552 entries from dependency cache.
NOTE: Resolving any missing task queue dependencies
ERROR: Nothing RPROVIDES 'docker' (but /home/jiwon/workspace/STM32MPU/Distribution-Package/azure/openstlinux-4.19-thud-mp1-19-10-09/layers/meta-st/meta-st-openstlinux/recipes-st/images/st-image-weston.bb RDEPENDS on or otherwise requires it)
NOTE: Runtime target 'docker' is unbuildable, removing...
Missing or unbuildable dependency chain was: ['docker']
ERROR: Required build target 'st-image-weston' has no buildable providers.
Missing or unbuildable dependency chain was: ['st-image-weston', 'docker']

Summary: There were 2 ERROR messages shown, returning a non-zero exit code.

```

**Comment by jwyune** [comment]
I was able to compile an image by adding these meta layers in **meta-openembedded**.

1. https://git.yoctoproject.org/git/meta-virtualization (Branch: Thud)
2. https://github.com/meta-rust/meta-rust.git (Commit ID: 9487b089ea4779c2b494b17b9254219226efa539)
3. https://github.com/Azure/meta-iotedge (Branch: Thud)

Then, I did `bitbake st-image-weston` and downloaded "FlashLayout_sdcard_stm32mp157a-dk1-basic.tsv" on an STM32MP157A-DK1. Nonetheless, I still see nothing related to Azure IoT Edge on the device. How can I fix this?
